packages/xml-views-validation/test/api-spec.ts

```diff
@@ -34,12 +34,47 @@ describe("the ui5 xml views validations API", () => {
     const { cst, tokenVector } = parse(xmlSnippet);
     const ast = buildAst(cst as DocumentCstNode, tokenVector);
 
-    const issues = validateXMLView({ model: ui5SemanticModel, xmlView: ast });
+    const issues = validateXMLView({
+      model: ui5SemanticModel,
+      xmlView: ast,
+      flexEnabled: false,
+    });
     expect(issues).to.have.lengthOf(2);
     const issueTypes = map(issues, (_) => _.kind);
     expect(issueTypes).to.include.members([
       "UnknownEnumValue",
       "UseOfDeprecatedClass",
     ]);
   });
+
+  it("will detect semantic UI5 xml view issues (smoke) when flexEnabled is true", () => {
```

Comment:
perhaps reduce this snippet to only test the stableID issue?
Do we need to duplicate large parts of the previous test?